assets/css/screen.css

```diff
@@ -1095,18 +1076,19 @@ The knock-on effect of this is ugly browser-scroll bars at the bottom, so 80px o
 
 .post-card-byline-date {
   font-size: 1.2rem;
-  color: #7b848f
+  color: #7b848f;
 }
 
-.single-author-byline {
-  display: flex;
-  flex-direction: column;
-  margin-left: 5px;
-  color: color(var(--midgrey) l(-10%));
-  font-size: 1.3rem;
-  line-height: 1.4em;
-  font-weight: 500;
-}
+/* NOT USED */
```

Comment:
should we remove the not used ones instead of commenting out?

Reply:
Will do 👍